libbeat/docs/upgrading.asciidoc

```diff
@@ -4,214 +4,142 @@
 
-NOTE: When upgrading to Metricbeat, keep in mind that any data you've collected
-with Topbeat is not compatible with the 5.0 version of the Beats dashboards
-because the underlying event data structures have changed.
+Starting with the 6.0 version, Filebeat and Metricbeat are moving to a directory
+layout for configuration, where each module is configured in its own
+configuration file.
 
-With the Beats 5.0 release, Topbeat is replaced by Metricbeat, which offers more
-functionality. More precisely, the Topbeat functionality is roughly equivalent
-to the "system" module in Metricbeat.
+While the all-in-one configuration is still fully supported, we recommend moving
+to the new layout at upgrade time. This typically means starting of with the new
+default configuration and modify it with the custom settings that you had in
+your old files.
 
-While there is no automatic way to convert a Topbeat configuration file to the
-equivalent of a Metricbeat configuration file, both configurations are simple
-enough that migrating shouldn't take long.
+===== New command `test config` command
 
-For example, this is the input section of the Topbeat configuration file:
+Beats 6.0 introduce a new command to test the configuration file, via the `test`
+command. For example:
 
-[source,yaml]
+[source,shell]
 ------------------------------------------------------------------------------
-input:
-  # In seconds, defines how often to read server statistics
-  period: 10
-
-  # Regular expression to match the processes that are monitored
-  # By default, all the processes are monitored
-  procs: [".*"]
-
-  # Statistics to collect (all enabled by default)
-  stats:
-    # per system statistics, by default is true
-    system: true
-
-    # per process statistics, by default is true
-    process: true
-
-    # file system information, by default is true
-    filesystem: true
-
-    # cpu usage per core, by default is false
-    cpu_per_core: false
+metricbeat test config
 ------------------------------------------------------------------------------
 
-And here is the equivalent configuration for the system module in Metricbeat:
+The old `-configtest` flag is still available, but deprecated.
 
-[source,yaml]
-------------------------------------------------------------------------------
-metricbeat.modules:
+===== Reference configuration files
 
-#------------------------------- System Module -------------------------------
-- module: system
-  metricsets:
-    # CPU stats
-    - cpu
+The `<beatname>.full.yml` file, which contains all the non-deprecated
+configuration options is renamed to `<beatname>.reference.yml` starting with
+Beats 6.0. We recommend using this file as a reference only, not for using it as
+the configuration file in production.
 
-    # System Load stats
-    - load
+==== Dashboard upgrades
 
-    # Per filesystem stats
-    - filesystem
+We recommend re-importing the Kibana dashboards after the Beats and Kibana
```

Comment:
I would say explicitly, that upgrading the Kibana doesn't require dashboards upgrade, but if the user is interested in the new version of the dashboards that are shipped with 6.0, then it can reimport them.

Reply:
I added a sentence to explain why we recommend that.